docs/source/preview_datasets.md

```diff
@@ -1,35 +1,48 @@
+1. **TablePreview:** For datasets returning tables/dataframes.
+2. **JSONPreview:** For datasets returning JSON objects.
+3. **PlotlyPreview:** For datasets returning Plotly JSON objects.
+4. **ImagePreview:** For datasets returning base64-encoded image strings.
 
-```{important}
-We recommend that you use the same version of Kedro that was most recently used to test this tutorial (0.19.0). To check the version installed, type `kedro -V` in your terminal window.
-```
+Whilst we currently support the above datasets. We are soon going to extend this functionality to other datasets. Users with custom datasets can also extend the preview functionality and we will cover that in the following sections. 
 
-In your terminal window, navigate to the folder you want to store the project. Generate the spaceflights tutorial project with all the code in place by using the [Kedro starter for the spaceflights tutorial](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pandas):
 
+**Preview Tabular Data**
 
-```bash
-kedro new --starter=spaceflights-pandas
-```
+The page titled [Preview Tabular Data in Kedro-viz](./preview_pandas_datasets.md) contains a spaceflight tutorial that explains how you can enable preview on Tabular datasets such as `pandas.CSVDataset` and `pandas.ExcelDataset`.
+
+**Preview Plotly Charts**
+
+The page titled [Preview Plotly charts in Kedro-viz](./preview_plotly_datasets.md) contains a spaceflight tutorial that explains how you can create interactive visualizations using `PlotlyDatasets` on Kedro-viz.
 
-When prompted for a project name, you can enter anything, but we will assume `Spaceflights` throughout.
+**Preview Matplotlib Charts**
 
-When your project is ready, navigate to the root directory of the project.
+The page titled [Preview Matplotlib charts in Kedro-viz](./preview_matplotlib_datasets.md) contains a spaceflight tutorial that explains how you can create static visualizations using `MatplotlibWriterDataset` on Kedro-viz.
+
+**Extend Preview to Custom Datasets**
+
+The page titled [Extend Preview to Custom Datasets](./preview_custom_datasets.md) contains information on how you can set up preview for custom datasets and what types are supported by Kedro-viz.
+
+```{toctree}
+:maxdepth: 1
+:hidden:
+preview_matplotlib_datasets
+preview_plotly_datasets
+preview_pandas_datasets
+preview_custom_datasets
+```
 
-## Configure the Data Catalog
 
-Kedro-Viz version 6.3.0 currently supports preview of two types of datasets:
 
-* `pandas.CSVDataset`
-* `pandas.ExcelDataset`
+## Disabling Previews
 
 
-To enable dataset preview, add the `preview_args` attribute to the kedro-viz configuration under the `metadata` section in the Data Catalog. Within preview_args, specify `nrows` as the number of rows to preview for the dataset.
+To disable dataset previews for specific datasets, you need to set preview: false under the kedro-viz key within the metadata section of your conf.yml file. Here's an example configuration:
```

Comment:
should we mention that previewing is made default from latest version of viz ? If someone is using old version of viz, it will be opt in ? Also, it will be a good idea to mention about the kedro-datasets version this new feature of kedro viz supports somewhere in the doc. Thank you

Reply:
good point.